packages/config/adders/official.ts

```diff
@@ -6,6 +6,7 @@ export const adderCategories: AdderCategories = {
 	css: ['tailwindcss'],
 	db: ['drizzle'],
 	additional: ['storybook', 'mdsvex', 'routify'],
+	baas: ['supabase'],
```

Comment:
```suggestion
	db: ['drizzle', 'supabase'],
	additional: ['storybook', 'mdsvex', 'routify'],
```
in accordance to Ben's comment below